src/engine/Engine.ts

```diff
@@ -971,6 +972,11 @@ module ex {
       private _update(delta: number) {
          if (this._isLoading) {
             // suspend updates untill loading is finished
+            this._loader.update(this, delta);
+            // Update input listeners
+            this.input.keyboard.update(delta);
```

Comment:
Ugh, yeah there really isn't a good solution for this right now. Loaders weren't designed to support all the ex functionality. Now that we are leveraging it it makes sense to maybe make Loaders part of a Scene.